client/src/AppBundle/Form/Admin/SatisfactionFilterType.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
```suggestion
<?php declare(strict_types=1);
```

(another thing to add to pre-commit hook in the future)